paneldb/parser/baitparser.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+
+
+def read_file(pathtofile):
+
```

Comment:
Add an extended comment (multi-line docstring) about what this function does, what is the input (format) and what it is returning. See here for a description : https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings
You could also take a look at some of our code from other projects.

Reply:
Tried to add it in the correct format